libs/community/tests/integration_tests/document_loaders/docker-compose/cratedb.yml

```diff
@@ -0,0 +1,20 @@
+version: "3"
+
+services:
+  postgresql:
+    image: crate/crate:nightly
```

Comment:
```suggestion
    image: crate:latest
```

Reply:
If someone might be running software tests using that configuration, sticking with `crate/crate:nightly` might be able to catch regressions early. On the other hand, it is not intended for production use. What do you think is applicable when submitting this to the upstream project?